src/treetensornetworks/projttns/projttn.jl

```diff
@@ -68,3 +68,11 @@ function make_environment(P::ProjTTN, state::AbstractTTN, e::AbstractEdge)
   )
   return P
 end
+
+function contract(P::ProjTTN)::ITensor
```

Comment:
```suggestion
function contract(P::ProjTTN)
```
Let's remove the other type assertions as well.

Also, why isn't this implemented?